.github/workflows/openqa.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: openQA - GitHub action example
+# yamllint disable-line rule:truthy
+on:
+  pull_request:
+  issue_comment:
+    types: [created, edited]
+  workflow_dispatch:
+env:
+  OPENQA_TOKEN: ${{ secrets.OPENQA_API_USER }}:${{ secrets.OPENQA_API_KEY }}:${{ secrets.OPENQA_API_SECRET }}
+  OPENQA_HOST: ${{ secrets.OPENQA_URL }}
+
+jobs:
+  trigger_and_monitor_openqa:
+    runs-on: ubuntu-latest
+    container:
+      image: registry.opensuse.org/home/okurz/container/ca/containers/tumbleweed:curl-jq-ssh-git
```

Comment:
discussed this in the SUSE QE Tools mob session 2023-03-09: When we want to make a github action likely we still want to have a container that just provides what is needed to trigger and monitor openQA jobs so we could:
1. Put tools/trigger-openqa-jobs into os-autoinst/scripts and build a container that contains everything
2. Put the monitor part into the client itself